lib/developer_portal/app/controllers/developer_portal/cms/new_content_controller.rb

```diff
@@ -54,7 +54,7 @@ def show
 
       # CMS::Page#mime_type is parsed and valid Mime::Type
       # in case it can't be parsed (nil, something invalid) returns 'text/html'
```

Comment:
This comment does not match [what it does](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#rendering-content-from-string) anymore.
`render :plain` returns `text/plain` and not `text/html` and `render :text` does. For that response type, the new way would be using `render :body`.

Reply:
But actually it should not be text/html as it is only text ...

Reply:
Oh you are right, render :body does not set the mime :)

I will change thanks!